Dockerfile

```diff
@@ -1,17 +1,10 @@
 FROM nginxinc/nginx-unprivileged:alpine
 
-ARG uriprefix
-ENV URIPREFIX=${uriprefix}
-
 WORKDIR /app
 COPY . .
 
 USER 0
-
 RUN chown -R 101 .
-RUN apk update && apk upgrade && apk add jq
```

Comment:
This was only needed for modifying the schemas files, but it increases consistency. It's not great practice to update & upgrade packages in a Docker image like this because it means every deployment could potentially have different versions of things outside of what nginx has released in their image